Production/src/MuonHitMatch.cc

```diff
@@ -47,7 +47,7 @@ void MuonHitMatch::CountMatches(const pat::Muon& muon, CountMap& n_matches)
 {
     for(const auto& segment : muon.matches()) {
         if(segment.segmentMatches.empty() && segment.rpcMatches.empty() && segment.gemMatches.empty() && segment.me0Matches.empty()) continue;
-        if(n_matches.count(segment.detector())) {
+        if(n_matches.count(segment.detector()) && segment.station()!=0) {
```

Comment:
I don't know if "0" is really valid (My understanding is insufficient to answer this). In order to fix the crash that happens otherwise [here](https://github.com/cms-tau-pog/TauMLTools/blob/master/Production/src/MuonHitMatch.cc#L29), the alternative would be to allow the station 0 [here](https://github.com/cms-tau-pog/TauMLTools/blob/master/Production/interface/MuonHitMatch.h#L16).

Reply:
I agree that the crush must be fixed. But it would be preferable to make a conscious choice whatever we want to ignore stations "0" or it could be useful, and then modify the code accordingly.